firmware/dvb-si2168-firmware/pkg.yaml

```diff
@@ -0,0 +1,34 @@
+name: dvb-si2168-firmware
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+steps:
+  - sources:
+       - url: https://github.com/OpenELEC/dvb-firmware/archive/refs/tags/{{ .LINUX_DVB_FIRMWARE }}.tar.gz
+         destination: dvb-firmware.tar.gz
+         sha256: cef3ce537d213e020af794cecf9de207e2882c375ceda39102eb6fa2580bad8d
+         sha512: 2372dba98083c76865f5f0f8101b1160888e03cdbe911dd08621e7b6f38e8a25ae5d56eefc21728bf65fd09ea613b53606df4021d3972b0bc9d2bd8b6cbe20a1
+    prepare:
+      - |
+        sed -i 's#$VERSION#{{ .VERSION }}#' /pkg/manifest.yaml
+      - |
+        tar -xvf dvb-firmware.tar.gz
+        rm dvb-firmware.tar.gz
+    install:
+      - |
+        mkdir -p /rootfs/lib/firmware
+        cp dvb-firmware-0.0.51/firmware/dvb-demod-si2168-02.fw /rootfs/lib/firmware
```

Comment:
shouldn't this copy all firmware?

Reply:
No because there's lot of non-needed firmware files included in the download.